app.js

```diff
@@ -154,6 +154,8 @@ function createPreviewWindow() {
 }
 
 function addDevToolsExtension() {
+  if (process.env.DEVTRON_DEV_TOOLS_PATH)
```

Comment:
What would be the value of `DEVTRON_DEV_TOOLS_PATH` if I decided to use it?

Reply:
that would be the node_modules path, as it is in the devDependencies:
`DEVTRON_DEV_TOOLS_PATH=./node_modules/devtron`

Reply:
It works! 😆